### ISSUE TITLE
VASP parsers not recognizing certain potentials

### ISSUE THREAD

**Issue by moritzh96** [nomad-coe/electronic-parsers#11]
I found that uploading OUTCAR files obtained with certain PAW potentials would result in some errors:

ERROR: Inconsistent number of ions and species. (parsers/vasp)
ERROR: len of atom position does not match number of atoms. (parsers/vasp)

I suspect that this might be caused by the name convention of these potentials (^PAW_PBE F_h_GW 20May2014) while the standard potentials are named (PAW_PBE F_h_GW 20May2014). Removing the "^" in the OUTCAR file, I encountered no errors.

I attached one of the problematic files
[OUTCAR.zip](https://github.com/nomad-coe/electronic-parsers/files/9849274/OUTCAR.zip)

**Comment by ndaelman-hu** [comment]
Dear Moritzh, thank you for posting this issue. We are currently adding pseudopotential support to the VASP parser. I am supervising the responsible PhD student. I'll reach out to you once this has been ammended.

**Comment by ndaelman-hu** [comment]
Dear Moritzh, I finally got around to looking into your issue.
I traced the error back to the pseudopotential name not being extracted: your example `^PAW_PBE F_h_GW 20May2014` starts with a non-letter character. This is easy to solve by relaxing the regex condition. I pushed a working solution to [11-vasp-parsers-not-recognizing-certain-potentials
](https://github.com/nomad-coe/electronic-parsers/tree/11-vasp-parsers-not-recognizing-certain-potentials)

@ladinesa I'm not even sure that we still need the `species` quantity. As far as I can tell, it only acts to extract `n_types` (which isn't even stored any more) and perform a sanity check. I now extract the same info, and more via the `PotParser`. If we still need the sanity check, we can perhaps use this data, or remove it all together.
